packages/client/src/runtime/core/protocol/common.ts

```diff
@@ -0,0 +1,27 @@
+import type { EngineQuery } from '@prisma/engine-core'
+
+import type { DMMF } from '../../dmmf-types'
+import { CallSite } from '../../utils/CallSite'
+import { MergedExtensionsList } from '../extensions/MergedExtensionsList'
+import { Action, JsArgs } from '../types/JsApi'
+
+export interface ProtocolMessage {
+  isWrite(): boolean
+  getBatchId(): string | undefined
+  toDebugString(): string
+  toEngineQuery(): EngineQuery
+  deserializeResponse(data: unknown, dataPath: string[]): unknown
+}
+
+export type CreateMessageOptions = {
+  action: Action
+  model?: string
```

Comment:
For consistency, could it be `modelName`, or rename `modelName` to model?

Reply:
Done